lib/ReactTags.js

```diff
@@ -190,7 +190,7 @@ var ReactTags = React.createClass({
     e.preventDefault()
     const clipboardData = e.clipboardData || window.clipboardData
     const string = clipboardData.getData('text')
-    string.split(/[ ,]+/).forEach((tag) => this.props.handleAddition(tag))
+    string.split(/[\s,]+/).forEach((tag) => this.props.handleAddition(tag))
```

Comment:
Can you add a comment on what this regex supposed to do? Will help developers save a Google search :)